hazelcast-client-legacy/src/main/java/com/hazelcast/client/spi/ProxyManager.java

```diff
@@ -305,13 +306,13 @@ private Connection getTargetOrOwnerConnection(final Address target) throws IOExc
     }
 
     public Address findNextAddressToSendCreateRequest() {
-        final int clusterSize = client.getClientClusterService().getSize();
+        int clusterSize = client.getClientClusterService().getSize();
         Member liteMember = null;
 
         final LoadBalancer loadBalancer = client.getLoadBalancer();
         for (int i = 0; i < clusterSize; i++) {
-            final Member member = loadBalancer.next();
-            if (!member.isLiteMember()) {
+            Member member = loadBalancer.next();
+            if (member != null && !member.isLiteMember()) {
                 return member.getAddress();
             } else if (liteMember == null) {
```

Comment:
if at first a lite member comes, than a null comes, than we can hit this `else` lite member as far as I understand.